Doc/library/contextlib.rst

```diff
@@ -29,6 +29,15 @@ Functions and classes provided:
    .. versionadded:: 3.6
 
 
+.. class:: AbstractAsyncContextManager
+
+   An :term:`abstract base class` similar to
```

Comment:
I'm wondering if we could just copy the paragraph from `AbstractContextManager` doc. "similar to" isn't easy to read here.